app/views/publishers/vacancies/edit_vacancy_sections/_edit_applying_for_the_job.html.slim

```diff
@@ -1,63 +1,43 @@
-        dd.app-check-your-answers__answer.first-question
-          = @vacancy.personal_statement
-    - if @vacancy.apply_through_teaching_vacancies == "no"
-      .app-check-your-answers__contents
-        dt.app-check-your-answers__question#application_link
-          h4.govuk-heading-s = t("jobs.application_link")
-        dd.app-check-your-answers__answer
-          - if @vacancy.application_link.present?
-            = govuk_link_to(@vacancy.application_link, @vacancy.application_link, "aria-label": t("jobs.aria_labels.apply_link"), target: "_blank", rel: "noopener noreferrer")
-          - else
-            = t("jobs.not_defined")
-
-  .app-check-your-answers__contents
-    dt.app-check-your-answers__question.first-question#contact_email
-      h4.govuk-heading-s = t("jobs.contact_email")
-    dd.app-check-your-answers__answer.first-question
-      = govuk_mail_to "Job contact email", @vacancy.contact_email, "aria-label": t("jobs.aria_labels.contact_email_link", email: @vacancy.contact_email)
-
-  .app-check-your-answers__contents
-    dt.app-check-your-answers__question#contact_number
-      h4.govuk-heading-s = t("jobs.contact_number")
-    dd.app-check-your-answers__answer
-      = @vacancy.contact_number.present? ? govuk_link_to(@vacancy.contact_number, "tel:#{@vacancy.contact_number}") : t("jobs.not_defined")
-
-  .app-check-your-answers__contents
-    dt.app-check-your-answers__question#school_visits
-      h4.govuk-heading-s = t("jobs.#{school_or_trust_visits(@vacancy.parent_organisation)}")
-    dd.app-check-your-answers__answer = @vacancy.school_visits.presence || t("jobs.not_defined")
-
-  - unless JobseekerApplicationsFeature.enabled?
-    .app-check-your-answers__contents
-      dt.app-check-your-answers__question#how_to_apply
-        h4.govuk-heading-s = t("jobs.how_to_apply")
-      dd.app-check-your-answers__answer = @vacancy.how_to_apply.presence || t("jobs.not_defined")
-
-    .app-check-your-answers__contents
-      dt.app-check-your-answers__question#application_link
-        h4.govuk-heading-s = t("jobs.application_link")
-      dd.app-check-your-answers__answer
-        - if @vacancy.application_link.present?
-          = govuk_link_to(@vacancy.application_link, @vacancy.application_link, "aria-label": t("jobs.aria_labels.apply_link"), target: "_blank", rel: "noopener noreferrer")
-        - else
-          = t("jobs.not_defined")
+= render Shared::ReviewComponent.new(id: "applying_for_the_job", title: t("jobs.applying_for_the_job"), edit_link: organisation_job_build_path(@vacancy.id, :applying_for_the_job))
+
+.review-component
+  .review-component__body
+    = render GovukComponent::SummaryList.new do |component|
+      - if JobseekerApplicationsFeature.enabled?
+        - component.slot(:row,
+          key: t("jobs.apply_through_teaching_vacancies"),
+          value: @vacancy.apply_through_teaching_vacancies&.capitalize)
+
+        / - if @vacancy.apply_through_teaching_vacancies == "yes"
```

Comment:
Why's it commented out?

Reply:
oops

Reply:
ah i remember, this is cause there is actually a bug here (unrelated to this PR i think) and i temporarily commented it out. will investigate more